xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -1545,6 +1545,14 @@ CDemuxStream* CDVDDemuxFFmpeg::AddStream(int streamIdx)
       return nullptr;
     }
 
+    // FFmpeg adds the image attachments as video streams
+    if ((pStream->disposition & AV_DISPOSITION_ATTACHED_PIC) != 0)
+    {
+      CLog::LogF(LOGDEBUG, "CDVDDemuxFFmpeg::AddStream - discarding image attachment");
```

Comment:
```suggestion
      CLog::LogF(LOGDEBUG, "discarding image attachment");
```
LogF already adds class and function name

Reply:
Done.